File: maratona/2968.py
n, m = map(int, input().split())

total = n * m
for i in range(1, 10):
    qnt = (total * (i / 10))
    if qnt % 1 != 0:
        qnt += 1

    qnt = int(qnt)
    if i != 9:
        print("%d" % qnt, end=' ')
    else:
        print("%d" % qnt, end='\n')

File: maratona/2968.cpp
#include <iostream>
#include <math.h>
using namespace std;


int main() {
	int v, n;
	cin >> v >> n;
	
	int total = v * n;
	
	for (int i = 1; i <= 9; i += 1) {
		double qnt = total * (i / 10.0);
		
		
		if (qnt - (trunc(qnt)) != 0.0)
			cout <<  (int)qnt + 1;
		else 	
			cout << (int)qnt;
		
		if (i != 9)
			cout << " ";
	}
	cout << endl;
	return 0;
}